src/components/Filter.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../index.scss';
+
+export const Filter = () => {
+  return (
+    <section className="filter">
```

Comment:
I'm not sure what the UI looks like, but should this be a `<ul>`? Is this a menu?

Reply:
Hm, this might be better as a UL. It's a list of buttons, but they are not in a list in the UI. Not sure if that makes a difference.

Reply:
Ah I see - I think these should be `radio inputs`, what do you think?

One is selected by default you have the option to select others, but not more than 1.

Reply:
totally didn't think you could have radio inputs as buttons. with the way I have it styled you can only choose one at a time.

Reply:
When you're looking at UI like this and deciding on how to build the components, you should always reach for native HTML5 components first. You have to ask yourself, `what is the intended behavior`?

Each HTML5 component is very specific and can handle most usecases.

That's how I know it should be a radio button because a radio group can have only 1 selected item at a time.

If multiple could be selected, we would use checkboxes instead.

After that, it's all just styling.

Yes, _these look like buttons, but how do they behave_ - that's the determining factor.